src/flamenco/runtime/tests/fd_exec_instr_test.c

```diff
@@ -315,8 +338,36 @@ _context_create( fd_exec_instr_test_runner_t *        runner,
   /* Add accounts to bpf program cache */
   fd_bpf_scan_and_create_bpf_program_cache_entry( slot_ctx, funk_txn );
 
-  /* Restore sysvar cache */
+  /* Fill missing sysvar cache values with defaults */
+  /* We create mock accounts for each of the sysvars and hardcode the data fields before loading it into the account manager */
+  /* We use Agave sysvar defaults for data field values */
+
+  /* Clock */
+  // https://github.com/firedancer-io/solfuzz-agave/blob/agave-v2.0/src/lib.rs#L466-L474
+  uchar clock_raw_data[40] = {10, 0, 0, 0, 0, 0, 0, 0, // slot = 10
+                              0, 0, 0, 0, 0, 0, 0, 0,  // epoch_start_timestamp = 0
+                              0, 0, 0, 0, 0, 0, 0, 0,  // epoch = 0
+                              0, 0, 0, 0, 0, 0, 0, 0,  // leader_schedule_epoch = 0
+                              0, 0, 0, 0, 0, 0, 0, 0}; // unix_timestamp = 0
```

Comment:
If it's not a big pain, do you mind constructing a new `fd_sol_sysvar_clock_t` instance and use the `fd_sol_sysvar_clock_encode` API to encode this into the raw data?

It'll be easier to both write and read and quicker to add new ones in the future.

If it's a pain to change don't worry :)